src/components/screens/wallet/votes/voteRow.js

```diff
@@ -27,7 +26,6 @@ const VoteRow = ({
         <div className={styles.accountInfo}>
           <span className={`${styles.title} vote-username`}>{data.username}</span>
           <span className="showOnLargeViewPort">{data.address}</span>
-          <span className="hideOnLargeViewPort">{data.address.replace(regex.lskAddressTrunk, '$1...$3')}</span>
```

Comment:
Why is this line removed? This will cause that the address won't be visible on small screensizes.

Reply:
This doesn't account for when there's no receipt. There were too many of such small issues caused by the new structure of vote tx. I'll fix these issues by #3074.